packages/api-client/src/components/CommandPalette/TheCommandPalette.vue

```diff
@@ -153,6 +161,21 @@ const openCommandPalette = ({
   nextTick(() => commandInputRef.value?.focus())
 }
 
+/** Focus on first result when conducting a search */
+watch(commandQuery, (newQuery) => {
+  if (newQuery && searchResultsWithPlaceholderResults.value.length > 0) {
+    selectedSearchResult.value = 0
+    nextTick(() => {
+      commandRefs.value[selectedSearchResult.value]?.scrollIntoView({
+        behavior: 'smooth',
+        block: 'center',
```

Comment:
Should this be `nearest` like the search modal?

Reply:
Nearest all the things